dsp-value-objects/src/test/scala/dsp/valueobjects/GroupSpec.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.valueobjects
+
+import dsp.valueobjects.Group._
+import zio.prelude.Validation
+import zio.test._
+
+/**
+ * This spec is used to test the [[Group]] value objects creation.
+ */
+object GroupSpec extends ZIOSpecDefault {
+  private val validName        = "Valid group name"
+  private val invalidName      = "Invalid group name\r"
+  private val validDescription = Seq(V2.StringLiteralV2(value = "Valid group description", language = Some("en")))
+  private val invalidDescription = Seq(
+    V2.StringLiteralV2(value = "Invalid group description \r", language = Some("en"))
+  )
+
+  def spec = (groupNameTest + groupDescriptionsTest + groupStatusTest + groupSelfJoinTest)
+
+  private val groupNameTest = suite("GroupSpec - GroupName")(
+    test("pass an empty value and throw an error") {
+      assertTrue(GroupName.make("") == Validation.fail(V2.BadRequestException(GroupErrorMessages.GroupNameMissing)))
+      assertTrue(
```

Comment:
I think multiple `assertTrue()` need to be combined with `&&`, otherwise only the last of them is tested

Reply:
could be tested by changing `==` to `!=` and seeing if the test still passes